### PR TITLE
fixed PHP7.1 compatibility issue

### PR DESCRIPTION
**Pull request by matweew**
Warning: Declaration of ActiveRecord\DateTime::setTime($hour, $minute, $second = 0) should be compatible with DateTime::setTime($hour, $minute, $second = NULL, $microseconds = NULL)

### CODE REVIEW

**Comment by bartveneman** [comment]
Hey cool, was just running into this last week :)

File: lib/DateTime.php [hunk 1]
```diff
@@ -159,13 +159,13 @@ public function setDate($year, $month, $day)
...
 	{
 		$this->flag_dirty();
 		return parent::setISODate($year, $week, $day);
 	}
 
-	public function setTime($hour, $minute, $second = 0)
+	public function setTime($hour, $minute, $second = null, $microseconds = null)
 	{
 		$this->flag_dirty();
 		return parent::setTime($hour, $minute, $second);
```
Comment (koenpunt) [inline lib/DateTime.php]: Shouldn't the argument be passed through?

**Review by koenpunt — COMMENTED** [review]
Maybe add php 7.1 to the Travis config

**Comment by koenpunt** [comment]
Oh nevermind, it's already in there

File: lib/DateTime.php (hunk 1 above)
Comment (Rican7) [inline lib/DateTime.php]: Bumping.

**Comment by shmax** [comment]
This has been fixed in this fork, which is actively maintained:

https://github.com/php-activerecord/activerecord

You can grab it from packagist:
composer require php-patterns/activerecord.

New issues and PRs are welcome.

File: lib/DateTime.php (hunk 1 above)
Comment (DacostaDesarrollo) [reply lib/DateTime.php]: Muchas gracias, funciono para mi

**Pull request closed without merging**